tsconfig.json

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
Not sure if I'm correct, but this could change quite a bit I think?

Just this should be enough?

```json
{
  "compilerOptions": {
    "target": "es6",
    "module": "commonjs",
    "outDir": "dist",
    "strict": true,
    "noImplicitAny": true,
    "moduleResolution": "node",
    "baseUrl": "./",
    "paths": {
      "*": ["node_modules/*", "src/types/*"]
    },
    "allowSyntheticDefaultImports": true,
    "forceConsistentCasingInFileNames": true
  },
  "include": ["src/**/*"],
  "exclude": ["node_modules", "**/*.spec.ts"]
}
```

Reply:
Yep, this is nice. The current config looks pretty much the same, just needs some cleaning. I see some extra keys here, will commit those as well